marketplace-service/src/main/java/com/axonivy/market/service/impl/ImageServiceImpl.java

```diff
@@ -18,15 +19,18 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
+import java.util.function.Supplier;
 
 @Service
 @Log4j2
 public class ImageServiceImpl implements ImageService {
 
   private final ImageRepository imageRepository;
+  private final FileDownloadService fileDownloadService;
 
-  public ImageServiceImpl(ImageRepository imageRepository) {
+  public ImageServiceImpl(ImageRepository imageRepository, FileDownloadService fileDownloadService) {
```

Comment:
just optional, should we consider to use @AllArgsConstructor of loombok to avoid extend this method next time?